packages/api/src/routes/sensorRoute.js

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright 2019, 2020, 2021, 2022 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+const express = require('express');
+const multer = require('multer');
+
+const SensorController = require('../controllers/sensorController');
+
+const storage = multer.memoryStorage();
+const upload = multer({ storage });
+
+const router = express.Router();
+
+router.get('/get_sensors', SensorController.getSensorsByFarmId());
```

Comment:
I think at the moment we only need an add sensors endpoint

Reply:
We can keep the functions in the controllers with some todo comments if we want, but I think we should get rid of routes so users can not access them.